src/components/CartItem.tsx

```diff
@@ -2,18 +2,18 @@ import {
   incrementQuantity,
   decrementQuantity,
   removeFromCart,
-  CartItems,
+  CartItem,
 } from '../store/CartSlice'
 import AddRoundedIcon from '@mui/icons-material/AddRounded'
 import RemoveRoundedIcon from '@mui/icons-material/RemoveRounded'
 import DeleteRoundedIcon from '@mui/icons-material/DeleteRounded'
 import { useAppDispatch } from '../store/Hooks'
 
 export interface CardItemPops {
-  item: CartItems
+  item: CartItem
 }
 
-function CartItem({ item }: CardItemPops) {
+function CartItems({ item }: CardItemPops) {
```

Comment:
This rename will conflict with another PR.

Reply:
done in another PR